src/ui/Toggle/index.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../styles/variables';
+
+.sendbird-input-toggle-button {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+
+  box-shadow: 2px solid red;
```

Comment:
box-shadow: red >  same for both themes? :curious:

Reply:
it's a legacy, let me remove it thanks